plutus-script-utils/plutus-script-utils.cabal

```diff
@@ -110,7 +110,7 @@ library
   if flag(defer-plugin-errors)
     ghc-options: -fplugin-opt PlutusTx.Plugin:defer-errors
 
-test-suite plutus-ledger-test
+test-suite plutus-ledger-test2
```

Comment:
There is another executable with the exact same name in the `plutus-ledger` package.
This is a problem because the new flake exports all Haskell executables by their "short" name (i.e. without the package name prefixed to them). 
This means that one will override the other. 
Please suggest a new name for this executable or the one in `plutus-ledger`